### PR TITLE
Adding fs tc 83575082

### PR DESCRIPTION
**Pull request by julpark-rh**
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ON12XE

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by hkadam134 — COMMENTED** [review]
Hi Julius,

Can you mount the exports and run some IOs on both the mounts?

File: tests/cephfs/cephfs_nfs/nfs_multiple_export_using_single_conf.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+        if tf1 and tf2:
+            log.info("export created successfully")
+        else:
+            raise CommandFailed("Failed to export configs")
+        return 0
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_nfs/nfs_multiple_export_using_single_conf.py:115]: Do we need to have cleanup for this script

**Comment by julpark-rh** [comment]
> Hi Julius,
> 
> Can you mount the exports and run some IOs on both the mounts?

added

File: tests/cephfs/cephfs_nfs/nfs_multiple_export_using_single_conf.py (hunk 1 above)
Comment (julpark-rh) [reply tests/cephfs/cephfs_nfs/nfs_multiple_export_using_single_conf.py:130]: added

**Review by AmarnatReddy — APPROVED** [review]

**Pull request merged**